examples/BertNewsClassification/requirements.txt

```diff
@@ -1,10 +1,10 @@
-torch>=1.6.0
```

Comment:
i guess it can still be `torch>=1.6.0`.

Reply:
Since torch==1.9.0 requires changes in example scripts, it might fail in older versions of torch.